rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -590,63 +627,134 @@ def _compute_default_label_features(
         ]
 
     def _create_label_data(
```

Comment:
This is just too big a function with the new code that is added. Can you please break it down into smaller functions?